source/extensions/filters/http/lua/lua_filter.cc

```diff
@@ -485,6 +430,81 @@ int StreamHandleWrapper::luaImportPublicKey(lua_State* state) {
   return 1;
 }
 
+FireAndForgetHttpWriter::FireAndForgetHttpWriter(Filter& filter) : filter_(filter) {}
```

Comment:
i don't think this class buys us a whole lot, how about just having a `NoopCallbacks` that we pass to `makeHttpCall`?

Reply:
Hum, true.

Reply:
Good idea. Pushed.